sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -369,25 +369,25 @@ def key_id(self):
 
     @property
     def secret_id(self):
-        # type: () -> str
-        """:rtype: str"""
+        # type: () -> Optional[Any]
```

Comment:
It looks like the [secret ID of a CertificateBundle](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_generated/v7_2/models/_models_py3.py#L240) should always be a string, so should this be `Optional[str]`?

Reply:
Good point! I didn't check CertificateBundle, in this case I think `Optional[str]` would make more sense.